lib/recurly/billing_info.php

```diff
@@ -2,20 +2,6 @@
 
 class Recurly_BillingInfo extends Recurly_Resource
 {
-  protected static $_writeableAttributes;
-
-  public static function init()
-  {
-    Recurly_BillingInfo::$_writeableAttributes = array(
-      'first_name','last_name','name_on_account','ip_address',
-      'address1','address2','city','state','country','zip','phone','vat_number',
-      'number','month','year','verification_value','start_year','start_month','issue_number',
```

Comment:
I'll call out that I'm dropping some unused fields here: `'start_year','start_month','issue_number',`